Deploy-CAM.ps1

```diff
@@ -2709,6 +2661,155 @@ function Write-Host-Warning() {
     Write-Host ("`n$message") -ForegroundColor Red
 }
 
+# Prompt update Radius Settings in the Keyvault
+function Set-RadiusSettings() {
+    Param(
+        [string]$VaultName
+    )
+    # Check current MFA settings
+    $isRadiusMfaEnabled = Get-AzureKeyVaultSecret `
+        -VaultName $VaultName `
+        -Name "enableRadiusMfa" `
+        -ErrorAction stop
+    $isRadiusMfaEnabled = ([bool]$isRadiusMfaEnabled.SecretValueText)
+
+    $currentRadiusHost = Get-AzureKeyVaultSecret `
+        -VaultName $VaultName `
+        -Name "radiusServerHost" `
+        -ErrorAction stop
+    $currentRadiusHost = ([string]$currentRadiusHost.SecretValueText)
+
+    $currentRadiusPort = Get-AzureKeyVaultSecret `
+        -VaultName $VaultName `
+        -Name "radiusServerPort" `
+        -ErrorAction stop
+    $currentRadiusPort = ([string]$currentRadiusPort.SecretValueText)
+
+    $currentRadiusSecret = Get-AzureKeyVaultSecret `
+        -VaultName $VaultName `
+        -Name "radiusSharedSecret" `
+        -ErrorAction stop
+    $currentRadiusSecret = ([string]$currentRadiusSecret.SecretValueText)
+
+    # Prompt for RADIUS configuration if RADIUS has not been already explicitly been disabled
+    if ($ignorePrompts -and ($enableRadiusMfa -eq $null)) {
+        $enableRadiusMfa = $isRadiusMfaEnabled
+    }
+    if ($enableRadiusMfa -eq $null) {
+        $currentRadiusSetting = "disabled"
+        if ($isRadiusMfaEnabled) {
+            $currentRadiusSetting = "enabled"
+        }
+        $changeRadiusSettings = (confirmDialog ([String]::Format("Radius Multi-Factor Authentication is currently {0}. Do you want to change Multi-Factor Authentication settings?", $currentRadiusSetting))) -eq 'y'
```

Comment:
Can't you just put $currentRadiusSetting directly in the dual-quotes quoted string without the format function?

Reply:
You're right, don't know why I did this